### DISCUSSION TITLE
Improved Vote

### DISCUSSION THREAD

**Discussion by TimHarrio** [Zrips/CMI#3021] [Feature Requests]
**Description of feature/improvement request:**

Hello then here is my proposal,

I use CMI to manage the rewards of my votes and I think it would be good to add the total voting placeholder by week / month / since always to know the total vote which was made on the server.

I also think it will be possible to give rewards to the player for all X total votes (like VoteParty but no need to buy the plugin in addition just for that)

**Comment by mrfloris** [comment]
If some of the basic features of VotingPlugin could be in the vote feature of CMI, then I have no reason to continue using votingplugin. The less plugins i run on my server the better. CMI's already getting pretty close. The way i use voting plugin isn't that complex.
